easysid.cpp

```diff
@@ -26,7 +41,15 @@ EXPORT void start_capture(const char *fn)
         delete wav;
         wav = NULL;
     }
-    wav = makePcmOutputWav(fn);
+    if (pcm == NULL) {
+        if (not initAudioOutput(DEFAULT_SAMPLING_RATE)) {
```

Comment:
It SHOULD.  This is only there to provide (currently) a pcm object, in which to output the sound to, IF you call start_capture() before calling set_sampling_parameters().  If you call it AFTER, then you're ok, because the capture file will use whatever sampling rate the current pcm object is using.